### DISCUSSION TITLE
CasaOS and Jellyfin server

### DISCUSSION THREAD

**Discussion by LITUATUI** [devmfc/debian-on-amlogic#19] [General]
As anybody tried to install CasaOS and Jellyfin server?

If yes, what's your experience?

I'm tempted to try it on a H96 Max W2 4/64 GB (S905W2 SoC) that costed me less than 20 euros on AliExpress.

**Comment by Bendy3192** [comment]
Hey, I tried only Jellyfin Server on my Tanix W2 (S905W2) and this works very well with my tv through DLNA, but I've tried only 720p and maybe 1080 (I don't remember exactly), Idk what will be with 4k content. I'm using external HDD via USB 3.0 and fast 32gb microSD card. My tv box specs maybe low-end, but my server using 300-400 mb when it is not heavily loaded. I have 2/16 specification. I think with 4/32 or more will be better experience if you want to something heavier like NAS or Nextcloud)

**Reply by markosc** [reply]
USB 3.0 or only USB 2.0 works for you ?

**Reply by devmfc** [reply]
These S905W2 boxes only have USB2 AFAIK. And IIRC, S905W2 doesn't have a USB3 block at all...
